pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -236,7 +250,7 @@ func (vm *windows) ConfigureKubeProxy(nodeName, hostSubnet string) error {
 		"--hostname-override=" + nodeName + " --kubeconfig=c:\\k\\kubeconfig " +
 		"--cluster-cidr=" + hostSubnet + " --log-dir=" + kubeProxyLogDir + " --logtostderr=false " +
 		"--network-name=OVNKubernetesHybridOverlayNetwork --source-vip=" + sVIP +
-		" --enable-dsr=false"
+		" --enable-dsr=false\""
```

Comment:
Why is this change needed? If at it is, then it needs to be in the previous commit.

Reply:
needed as the depend args are being added, moved to previous commit.